<?xml version="1.0"?>
<launch>
    <arg name="max_data" default="100" />
    <arg name="package_name" default="6dof_pose_estimator" />
    <arg name="object_name" default="HV8" />
    <arg name="dataset_name" default="data1" />

    <node pkg="6dof_pose_estimator" type="voxelization" name="voxelization" output="screen" >
        <param name="max_data" value="$(arg max_data)" />
        <param name="package_name" value="$(arg package_name)" />
        <param name="object_name" value="$(arg object_name)" />
        <param name="dataset_name" value="$(arg dataset_name)" />
    </node>
</launch>
